mage_ai/io/redshift.py

```diff
@@ -225,16 +225,24 @@ def __process():
                     cur.execute(query)
                 else:
                     if should_create_table:
-                        col_with_types = ', '.join(
-                            [f'{col} {col_type}' for col, col_type in columns_with_type],
-                        )
+                        if overwrite_type is not None:
+                            user_mod_columns, col_with_usr_types = get_user_type(overwrite_type)
+
+                        col_with_types = [f'{col} {col_type}' for col, col_type in columns_with_type
```

Comment:
Let's not change the order of the columns in the create table command.
Same in other databases.